lxd/api_cluster.go

```diff
@@ -335,6 +335,10 @@ func clusterPutBootstrap(d *Daemon, r *http.Request, req api.ClusterPut) respons
 
 		address := config.HTTPSAddress()
 
+		if util.IsWildCardAddress(address) {
+			return fmt.Errorf("cluster.https_address cannot copy wildcard core.https_address: %s", address)
```

Comment:
Couple of things here:

1. Maybe something like "Cannot use core.https_address %q for cluster.https_address as is wildcard address, please set cluster.https_address separately or set core.https_address to non-wildcard address" would be clearer.
2. You can use `%q` for quoting strings to separate error from problem value, saving the need for the colon.